charts/s1-agent/templates/common/configmaps.yaml

```diff
@@ -0,0 +1,40 @@
+{{ if .Values.configuration.env.helper.agent_injection_enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "agentInjection.name" . }}
+  labels:
+    {{- include "sentinelone.agent.labels" . | nindent 4 }}
+data:
+  agentconfig.yaml: |
+    containers:
+    - name: agent
+{{- if .Values.secrets.imagePullSecret }}
+      imagePullSecrets:
+        - name: {{ .Values.secrets.imagePullSecret }}
+{{- end }}
+      image: "{{ include "agent.full_url" . }}"
+      imagePullPolicy: {{ default "IfNotPresent" .Values.configuration.imagePullPolicy }}
+      securityContext:
+        runAsUser: 0
+        runAsGroup: 0
```

Comment:
why are we running as root?

Reply:
Admin capabilities are not allowed on fargate so we must run as root to access the monitored container filesystem.